### PR TITLE
[AN-5762] Delete failed video recordings

### PR DESCRIPTION
**Pull request by CalumMcCall**
#### APK
[Download build #10269](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10269/artifact/build/artifact/wire-dev-PR1414-10269.apk)
[Download build #10277](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10277/artifact/build/artifact/wire-dev-PR1414-10277.apk)
[Download build #10278](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10278/artifact/build/artifact/wire-dev-PR1414-10278.apk)
[Download build #10279](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10279/artifact/build/artifact/wire-dev-PR1414-10279.apk)
[Download build #10303](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10303/artifact/build/artifact/wire-dev-PR1414-10303.apk)
[Download build #10327](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10327/artifact/build/artifact/wire-dev-PR1414-10327.apk)
[Download build #10338](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10338/artifact/build/artifact/wire-dev-PR1414-10338.apk)
[Download build #10385](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10385/artifact/build/artifact/wire-dev-PR1414-10385.apk)

### CODE REVIEW

**Comment by deanrobertcook** [comment]
build again

File: app/src/main/scala/com/waz/zclient/WireApplication.scala [hunk 1]
```diff
@@ -194,6 +197,19 @@ object WireApplication {
       */
     bind [UiTrackingController]    to new UiTrackingController()
   }
+
+  protected def clearOldVideoFiles(context: Context): Unit = {
+    Option(context.getExternalCacheDir).foreach { _.listFiles().foreach { file =>
+      val fileName = file.getName
+      val cal = Calendar.getInstance
+      cal.add(Calendar.DAY_OF_YEAR, -7)
+      val fileModified = Calendar.getInstance()
```
Comment (makingthematrix) [inline app/src/main/scala/com/waz/zclient/WireApplication.scala]: You can take these two lines out of the loop.

**Comment by deanrobertcook** [comment]
build again

**Comment by CalumMcCall** [comment]
build again

**Comment by deanrobertcook** [comment]
@CalumMcCall you have a few PMD warnings

**Comment by CalumMcCall** [comment]
build again

File: app/src/main/java/com/waz/zclient/pages/main/conversation/AssetIntentsManager.java [hunk 2]
```diff
@@ -193,19 +183,17 @@ private static File getOutputMediaFile(IntentType type) {
     }
 
     public enum IntentType {
-        UNKOWN(-1, -1),
-        GALLERY(9411, 8411),
-        SKETCH_FROM_GALLERY(9416, 8416),
-        VIDEO(9412, 8412),
-        CAMERA(9413, 8413),
-        FILE_SHARING(9414, 8414);
+        UNKOWN(-1),
```
Comment (makingthematrix) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/AssetIntentsManager.java]: Since you're working on this file anyway, please fix the typo.

**Pull request merged**